HopsanGUI/PlotWindow.cpp

```diff
@@ -962,72 +962,76 @@ void PlotWindow::createPlotWindowFromTab()
 
 
 //! @todo should not run code on non bodeplot tabs
-void PlotWindow::createBodePlot(SharedVectorVariableT var1, SharedVectorVariableT var2, int Fmax)
+void PlotWindow::createBodePlot(SharedVectorVariableT var1, SharedVectorVariableT var2, int Fmax, bool bode, bool nyquist)
```

Comment:
OK, I think I understand. The Fmax parameter is in Hz. That is why there is multipliation and division with 2PI in various places. Could perhaps add a documentation block for this and the createNyquistPlot functions stating the intended unit.